tests/python_on_whales/components/test_system.py

```diff
@@ -23,6 +24,20 @@ def test_info():
     assert "local" in info.plugins.volume
 
 
+def test_events():
+    name = random_name()
+    docker.run("hello-world", remove=True, name=name)
+    # Takes some time for events to register
+    sleep(1)
+    events = docker.system.events(filters=[f"container={name}"])
+    # Check that we capture all the events from container create to destroy
+    assert len(events) >= 5
+    actions = []

```

Comment:
```suggestion
    actions = set()
```
actions can be a set, there is no need to use a list first and then convert it to a set

Reply:
Done